module/applications/actor/character-sheet-2.mjs

```diff
@@ -542,6 +543,35 @@ export default class ActorSheet5eCharacter2 extends ActorSheetV2Mixin(ActorSheet
 
   /* -------------------------------------------- */
 
+  /**
+   * Handle removing a pending trade from a facility.
+   * @param {Item5e} facility  The target facility.
+   * @returns {Promise<void|Item5e>}
+   * @protected
+   */
+  async _onRemovePendingTrade(facility) {
+    const result = await foundry.applications.api.DialogV2.confirm({
```

Comment:
Total aside, feels like there should be a system-implementation of `DialogV2`. Probably not doing more than just adding some of the styling.

Reply:
Yep, unfortunately I already used the name `Dialog5e` for a slightly different V2 dialog, so we'll have to find a new name :(

Reply:
This is why we can't have nice things.